packages/pyright-internal/src/languageService/completionProviderUtils.ts

```diff
@@ -93,8 +94,19 @@ export function getTypeDetail(
                     }
                 }
             }
-
-            return name + ': ' + evaluator.printType(type, { expandTypeAlias });
+            // handle the case where the type is a function is assigned to a variable
+            if (type.category === TypeCategory.OverloadedFunction || type.category === TypeCategory.Function) {
+                return getToolTipForType(
+                    type,
+                    /*label*/ '',
```

Comment:
Nit: Include spaces around the parameter name in the comment for consistency with the rest of the code base.